src/type/js/honeycomb.type.font.js

```diff
@@ -29,5 +31,5 @@ Honeycomb.Font = (function($) {
 })(jQuery);
```

Comment:
That's a great point David. If the font was included in the module, rather than requested in, would that work for all/majority of our apps?